modules/b2b_entities/dlg.c

```diff
@@ -964,8 +983,16 @@ int b2b_prescript_f(struct sip_msg *msg, void *uparam)
 				str ok = str_init("OK");
 
 				if(method_value == METHOD_BYE)
+				{
 					tmb.t_reply(msg, 200, &ok);
-				else
+					tm_tran = tmb.t_gett();
+					if(tm_tran && tm_tran!=T_UNDEFINED)
+					{
+						// Tracer callback: our reply on BYE
+						if (dlg && dlg->tracer_param)
```

Comment:
Here the tracing is also broken, as dlg is NULL all the time, we are on a `if(dlg== NULL)` branch